### DISCUSSION TITLE
Ability to enable entry of a specified site using GQL

### DISCUSSION THREAD

**Discussion by dande-sun** [craftcms/cms#11221] [Ideas] [answered]
I created an entry with GraphQL, but it's only enabled for primary site, so I need enable it for other sites. I tried this, it's not working.
![image](https://user-images.githubusercontent.com/93630861/168285899-ded28213-f532-4ae0-a20a-9280b7c87534.png)

**Comment by brandonkelly** [comment, marked as answer]
It’s not currently possible to edit elements’ site-specific status with GraphQL mutations. See #10830 for a related discussion.
